noir-projects/aztec-nr/aztec/src/state_vars/shared_mutable/scheduled_value_change.nr

```diff
@@ -133,19 +134,28 @@ impl<T> ScheduledValueChange<T> {
     }
 }
 
-impl<T> Packable<3> for ScheduledValueChange<T>
+impl<T, let N: u32> Packable<2 * N + 1> for ScheduledValueChange<T>
 where
-    T: ToField + FromField,
+    T: Packable<N>,
```

Comment:
This needs an explanation (i.e. we store 2 T's and one extra field for the block of change).

Reply:
https://github.com/AztecProtocol/aztec-packages/pull/11716/commits/806e6470a6c1dc79dfaae309e132241dad59a621